permissions/Phone.permission

```diff
@@ -49,3 +49,6 @@ dbus-user.broadcast com.jolla.PinQuery=com.jolla.PinQuery.*@/*
 # BEG sessionbus-com.jolla.settings
 dbus-user.call com.jolla.settings=com.jolla.settings.ui.*@/*
 # END sessionbus-com.jolla.settings
+
+# Contacts service
+dbus-user.talk com.jolla.contacts.ui
```

Comment:
I wonder if this should be in Contacts.permission or even in voicecall-ui.profile.

Reply:
Yeap, Contacts.permission file feels more logical home for this.